receiver/apachesparkreceiver/scraper.go

```diff
@@ -262,6 +262,8 @@ func (s *sparkScraper) recordStages(stageStats []models.Stage, now pcommon.Times
 			s.mb.RecordSparkStageStatusDataPoint(now, 0, false, false, true, false)
 		case "FAILED":
 			s.mb.RecordSparkStageStatusDataPoint(now, 0, false, false, false, true)
+		case "SKIPPED":
+			continue
```

Comment:
should we record it then?

Reply:
The problem is that SKIPPED is not a real record.
It means that this was already done.

So adding it as a record actually misleading.

Reply:
https://stackoverflow.com/questions/34580662/what-does-stage-skipped-mean-in-apache-spark-web-ui#:~:text=What%20does%20it%20mean%20by%20skipped%3F%20Typically%20it,shows%20that%20the%20next%20stage%20requires%20shuffling%20%28reduceByKey%29.

Reply:
Thank you for your clarification.